src/custom-elements/parts-kit.tsx

```diff
@@ -1,31 +1,86 @@
 import { render } from 'preact'
 import { App } from '../app.tsx'
+// https://vitejs.dev/guide/features.html#disabling-css-injection-into-the-page
+import cssString from '../index.css?inline'
+
+/**
+ * Ensure that we only set styles once
+ */
+let hasSetUpStyles = false
 
 export default class PartsKit extends HTMLElement {
+  private shadow: ShadowRoot
+
+  /**
+   * Using constructable stylesheets.
+   *
+   * @see https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM?ref=blog.jonas.liljegren.org#constructable_stylesheets
+   *
+   * In the future, we could make this variable globally shared across all web components that want to load our
+   * Tailwind based stylesheet.
+   *
+   * This doesn't work in Safari 16.3 and less. 16.4 was Released 2023-03-27
+   */
+  private styleSheet = new CSSStyleSheet()
```

Comment:
I was [reading more](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM?ref=blog.jonas.liljegren.org#choosing_between_programmatic_and_declarative_options) about constructable stylesheets. They seem really cool.

When paired with custom elements, you can have a single `CSSStyleSheet()` shared across each element.

Any update to the stylesheet will be reflected across all the custom elements.

I could see this being particularly useful if we end up making some custom elements that can be used within each part demo.

A `<parts-kit-markdown>`  tag could use this projects Tailwind.

Or even re-creating [Blueprints ComponentPreview](https://github.com/vigetlabs/blueprint/blob/main/src/components/app/ComponentPreview.astro) as a custom element that we could use across WP, Craft, Etc.